src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -504,7 +517,9 @@ export class InlineChatWidget {
 		this.updateInfo('');
 
 		this._elements.accessibleViewer.classList.toggle('hidden', true);
-
+		this._codeBlockModelCollection.clear();
+		this._responseViewModel = undefined;
```

Comment:
this should be part of the `_chatMessageDisposables`-store

Reply:
it is so an delete this